.github/workflows/ci-pytorch-test-full.yml

```diff
@@ -95,14 +95,17 @@ jobs:
     - name: Pull legacy checkpoints
       run: bash .actions/pull_legacy_checkpoints.sh
 
+    - name: Adjust PyTorch versions in requirements files
+      if: matrix.requires != 'oldest'
```

Comment:
It will do that even in the oldest job, because pytorch-version will be 1.9

Reply:
if we will keep then align, but if miss it we would have silently installed different version

Reply:
Such misalignment should be covered by 

```python
        python -c "import torch; assert torch.__version__.startswith('${{ matrix.pytorch-version }}')"
```

in line 146

Reply:
yes, unless you push it to be aligned by this code block:
```
python ./requirements/pytorch/adjust-versions.py requirements/pytorch/base.txt ${{ matrix.pytorch-version }}
```
so in other words it negates the min replacement, right?

Reply:
It does not negate anything. If you run locally

```py
python .actions/assistant.py replace_oldest_ver
python ./requirements/pytorch/adjust-versions.py requirements/pytorch/base.txt 1.9
git diff requirements/pytorch/base.txt
```

You'll see it correctly sets

```python
torch==1.9.1
```

Reply:
Yes, if this CI PT version is identical to min version in requirements, but the situation I want to prevent is that we update requirements and miss this CI that the CI would be running the wrong version; see for example:
```
python .actions/assistant.py replace_oldest_ver
python ./requirements/pytorch/adjust-versions.py requirements/pytorch/base.txt 1.10
git diff requirements/pytorch/base.txt
```
But in principle, the situation is requirements are updated to >=1.11, but CI still say 1.9, then we would run CI with 1.9 instead of 1.11

Reply:
this si really just prevention issues cause by human in the loop :otter:

Reply:
Okay. Resolving